pkgs/development/python-modules/deap/default.nix

```
@@ -9,19 +18,18 @@ buildPythonPackage rec {
     hash = "sha256-zAHemJLfp9G8mAPasoiS/q0XfwGCyB20c2CiQOrXeP8=";
   };
 
-  propagatedBuildInputs = [ numpy matplotlib ];
+  propagatedBuildInputs = [ matplotlib numpy pytest ];
 
-  nativeCheckInputs = [ nose ];
   checkPhase = ''
-    nosetests --verbosity=3
+    runHook preCheck
+    ${python3.interpreter} -m unittest
+    runHook postCheck
   '';
 
   meta = with lib; {
     description = "DEAP is a novel evolutionary computation framework for rapid prototyping and testing of ideas.";
```

```suggestion
    description = "A novel evolutionary computation framework for rapid prototyping and testing of ideas";
```
Could you please update this?
https://nixos.org/manual/nixpkgs/stable/#var-meta-description

---

Wonder how that was merged before haha